torch_geometric/nn/glob/__init__.py

```diff
@@ -35,3 +27,48 @@
 class GlobalAttention(AttentionalAggregation):
     def __call__(self, x, batch=None, size=None):
         return super().__call__(x, batch, dim_size=size)
+
+
+sum_aggr = SumAggregation()
+
+
+@deprecated(
+    details="use 'nn.aggr.SumAggregation' instead",
+    func_name='nn.glob.global_add_pool',
+)
+def global_add_pool(x, index, size):
+    return sum_aggr(x, index, dim_size=size)
+
+
+mean_aggr = MeanAggregation()
+
+
+@deprecated(
+    details="use 'nn.aggr.MeanAggregation' instead",
+    func_name='nn.glob.global_mean_pool',
+)
+def global_mean_pool(x, index, size):
+    return mean_aggr(x, index, dim_size=size)
+
+
+max_aggr = MaxAggregation()
+
+
+@deprecated(
+    details="use 'nn.aggr.MaxAggregation' instead",
+    func_name='nn.glob.global_max_pool',
+)
+def global_max_pool(x, index, size):
+    return max_aggr(x, index, dim_size=size)
+
+
+@deprecated(
```

Comment:
We can safely drop this. It just got recently added to master. I believe we can just replace its usage within the PyG code-base with one of the aggregator modules from `nn.aggr`.

Reply:
Okay. I'll go through and depreciate the use of this. 👍